app/src/main/java/fr/free/nrw/commons/upload/DescriptionsAdapter.java

```diff
@@ -106,6 +116,7 @@ public void init(int position) {
                 descItemEditText.setText("");
             }
             if (position == 0) {
+                removeDescriptionImageView.setVisibility(View.INVISIBLE);
```

Comment:
it also needs a paired `setVisibility(VISIBLE)` if this view gets recycled

Reply:
I am unable to understand 😅 
by default, the visibility of all image views are set to VISIBLE so I am setting it INVISIBLE only for the position 0

Reply:
This is in an Adapter which preforms view recycling, because creating views is expensive a list is actually about 10 views reused over and over again instead of inflating theoretical thousands of views. Read more [here](https://developer.android.com/guide/topics/ui/layout/recyclerview#structure).

If you have enough views the view at position 0 is going to come back again further down the list and the removeImageView will be invisible because you set it here.

Reply:
@macgills I understand your point now.

For my understanding of recycler views, when we scroll the position is something like-
```
0
1
2
3
0
1
2
3
0
1
2
3
```

So you are correct when the position again becomes 0 the element down the list will have the invisible "x"

but my question is if we have a large list of views and we scroll down then why the "i" icon isn't repeated?

Reply:
check the else branch of position ==0

Just because it is the same view being used at position 5 (viewo) it is using the data of position5 to render